cli/src/commands/upload.command.ts

```diff
@@ -8,7 +8,7 @@ import { BaseCommand } from './base-command';
 import { basename } from 'node:path';
 import { access, constants, stat, unlink } from 'node:fs/promises';
 import { createHash } from 'node:crypto';
-import Os from 'os';
+import Os from 'node:os';
```

Comment:
I'd expect this to be `os`. I wasn't going to mention it since it's unrelated, but then I saw it imported as `os` in the next file in this PR